### DISCUSSION TITLE
How can this be used in a Jenkins pipeline?

### DISCUSSION THREAD

**Discussion by jreimone** [green-coding-solutions/eco-ci-energy-estimation#85] [Q&A]
Hello again 👋,
your awesome tool can be used on Github and Gitlab pipelines. Is it also possible to use it on Jenkins?

All the best,
Jan

**Comment by ArneTR** [comment]
Hey Jan,

we have used it with one client in a Jenkins Freestyle Project. Since we just issue CLI calls effectively it just blends into the sequential commands you use there anyway.

An example how simple CLI commands work with the tool are in the local test here: https://github.com/green-coding-solutions/eco-ci-energy-estimation/blob/main/local_ci.example.sh

Another option we have so far not investiaged is producing a Jenkins Plugin. If that is interesting for you we are happy to help here and would be love to take a PR :)
